src/content/changes/highlights/2022-11-17.md

```diff
@@ -0,0 +1,14 @@
+---
+date: 2022-11-17T10:00:00
+title: Highlights for the week ending November 17, 2022
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Apps
+
+- [falco-app](https://github.com/giantswarm/falco-app) version [v0.4.1](https://github.com/giantswarm/falco-app/blob/main/CHANGELOG.md#041---2022-11-16) makes it possible to use the Vertical Pod Autoscaler (VPA) to manage Falco resources (off by default).
+- [kyverno-policies](https://github.com/giantswarm/kyverno-policies) version [v0.18.0](https://github.com/giantswarm/kyverno-policies/blob/main/CHANGELOG.md#0180---2022-11-16) updates to upstream kyverno-policies version 1.7.5.
+
```

Comment:
## User interfaces

- [kubectl-gs v2.28.2](https://github.com/giantswarm/kubectl-gs/releases/tag/v2.28.2): The `kubectl-gs login` command no longer writes to the main kubeconfig file in case there are no changes in access tokens and/or the current context. Also setting the `--context` flag will no longer affect the current context (`current-context` in kubeconfig).

Reply:
suggestion integrated